_posts/en/newsletters/2022-05-25-newsletter.md

```diff
@@ -0,0 +1,193 @@
+  alternative designs that were considered by found deficient, and links
+  to a [presentation][zhao preso] with additional details.
+
+- **Miner extractable value discussion:** developer /dev/fd0
+  [posted][fd0 ctv9] to the Bitcoin-Dev mailing list a summary of the
+  [ninth IRC meeting][ctv9] about [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] (CTV).  Among other topics discussed during
+  that meeting, Jeremy Rubin listed several concerns he's heard people
+  express about recursive [covenants][topic covenants] (which CTV does
+  not enable).  One of those concerns was creating Miner Extractable
+  Value (MEV) significantly beyond the amount available from running a
+  simple transaction selection algorithm such as that provided by
+  Bitcoin Core.
+
+    MEV has become a particular concern in Ethereum and related
+    protocols where the use of public onchain trading protocols allows
+    miners to frontrun trades.  For example, imagine the following two
+    unconfirmed transactions both being available to mine in the next
+    block:
+
+    * Alice sells *x* to Bob for 1 ETH
+    * Bob sells *x* to Carol for 2 ETH (Bob earns 1 ETH in profit)
+
+    <br>If those two exchanges are conducted using a public onchain trading
+    protocol, a miner can cut Bob out of the transaction.  For example:
+
+    * Alice sells *x* to Miner Mallory for 1 ETH
+    * Miner Mallory sells *x* to Carol for 2 ETH (Mallory earns 1 ETH in
+      profit; Bob earns nothing)
+
+    <br>Obviously this is a problem for Bob, but it also creates several
+    problems for the network.  The first problem is that, in the above
+    example, Miner Mallory needs to have 1 ETH in a wallet in order to
+    exploit the situation.  Every other miner attempting to mine the
+    next block also needs to have their own 1 ETH.  That wallet value is
+    independent of each miner's hashrate, so two miners can join
+    together to halve the amount of money they each need to store in
+    their wallets for capturing MEV---producing an economy of scale that
+    encourages centralization of mining and which can leave the network
+    more vulnerable to transaction censorship.  A [report][bitmex
+    flashbots] by BitMex Research claims that a centralized service that
+    brokers these types of MEV transactions was being used by 90% of
+    Ethereum hashrate at the time the report was written.  To obtain the
+    maximum returns, that service could be changed to discourage mining
+    competing transactions, which would effective give it the power to
+    censor transactions if it was used by 100% of miners (or if it was
+    used by more than 50% of miners and allowed to engage in block chain
+    reorganization).
+
+    A second problem occurs if the original transaction from Alice to
+    Bob is confirmed in a block before the transaction from Bob to Carol
+    becomes available.  Now there's a 1 ETH bounty for any miner who
```

Comment:
this is sort of confusing and not really the problem.

The issue is that Mallory's modified txn is also reorgable by Eve.